File: java/onechicken.java
import java.util.Scanner;

public class onechicken {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int N = scan.nextInt();
		int M = scan.nextInt();
		int diff = M - N;
		if (diff > 0)
			System.out.printf("Dr. Chaz will have %s piece%s of chicken left over!", diff, diff > 1 ? "s" : "");
		else
			System.out.printf("Dr. Chaz needs %s more piece%s of chicken!", -diff, -diff > 1 ? "s" : "");
		scan.close();
	}
}

File: ts/src/onechicken.ts
import readline = require('readline')

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    const [peoples, chickens] = line.split(' ').map(Number)
    const diff = (chickens ?? 0) - (peoples ?? 0)
    let str = 'Dr. Chaz needs %s more piece%s of chicken!'
    if (diff > 0) {
      str = 'Dr. Chaz will have %s piece%s of chicken left over!'
    }
    console.log(str, Math.abs(diff), Math.abs(diff) > 1 ? 's' : '')
  })
  .on('close', () => {
    process.exit(0)
  })
